x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -308,6 +309,1604 @@ export const registerCollector: RegisterCollector = ({
+                  _meta: { description: 'The number of partial failure rules' },
+                },
+                top_partial_failure: {
+                  type: 'array',
+                  items: {
+                    message: {
+                      type: 'keyword',
+                      _meta: { description: 'Failed rule message' },
+                    },
+                    count: {
+                      type: 'long',
+                      _meta: { description: 'Number of times the message occurred' },
+                    },
+                  },
+                },
+                succeeded: {
+                  type: 'long',
+                  _meta: { description: 'The number of successful rules' },
+                },
+                index_duration: {
+                  max: {
+                    type: 'float',
+                    _meta: { description: 'The max duration' },
+                  },
+                  avg: {
+                    type: 'float',
+                    _meta: { description: 'The avg duration' },
+                  },
+                  min: {
+                    type: 'float',
+                    _meta: { description: 'The min duration' },
+                  },
+                },
+                search_duration: {
+                  max: {
+                    type: 'float',
+                    _meta: { description: 'The max duration' },
+                  },
+                  avg: {
+                    type: 'float',
+                    _meta: { description: 'The avg duration' },
+                  },
+                  min: {
+                    type: 'float',
+                    _meta: { description: 'The min duration' },
+                  },
+                },
+                gap_duration: {
+                  max: {
+                    type: 'float',
+                    _meta: { description: 'The max duration' },
+                  },
+                  avg: {
+                    type: 'float',
+                    _meta: { description: 'The avg duration' },
+                  },
+                  min: {
+                    type: 'float',
+                    _meta: { description: 'The min duration' },
+                  },
+                },
+                gap_count: {
+                  type: 'long',
+                  _meta: { description: 'The count of gaps' },
+                },
+              },
+              total: {
+                failed: {
+                  type: 'long',
+                  _meta: { description: 'The number of failed rules' },
+                },
+                partial_failure: {
+                  type: 'long',
+                  _meta: { description: 'The number of partial failure rules' },
+                },
+                succeeded: {
+                  type: 'long',
+                  _meta: { description: 'The number of succeeded rules' },
+                },
+              },
+            },
+            elastic_rules: {
```

Comment:
Interesting that this (and custom_rules) key is peers with all the other rules types when this technically isn't its own `rule type`, but I guess these are just 'categories of rules', and so not just limited to `types`. Like we could have a `rule_enabled_for_over_a_month` category if we wanted and sub-divide all these same metrics. All good! 👍